client/src/utils/attributeAndParamUtils.js

```diff
@@ -501,6 +501,25 @@ const initSsotSessionStorage = (robotId) => {
         console.error(error);
       });
 };
+
+/**
+ * @description Will send a backend call to delete a robot
+ * @param {String} robotId Id of the robot that is deleted
+ */
+const deleteRobotFromDB = async (robotId)  => {
+  const requestStringParameters = `/ssot/delete/${robotId}`;
+  const deleteMethod = {
+    method: 'DELETE',
+  };
+  await fetch(requestStringParameters, deleteMethod)
+    .then(() => {
+      console.log('removed');
+    })
```

Comment:
Please don't log this line. If you want to display a notification to the user use customNotification().

Reply:
+1, but customNotification is not available on this branch yet